ifrs17/Test/ImportStorageTest.ipynb

```diff
@@ -1165,7 +1165,10 @@
                 "\n                                                   sampleGic with {SystemName = \"Gross2\", LiabilityType = LiabilityTypes.LRC, ContractualCurrency = \"USD\", ValuationApproach = \"BBA\"},",
                 "\n                                                   sampleGic with {SystemName = \"Gross3\", LiabilityType = LiabilityTypes.LRC, ContractualCurrency = \"USD\", ValuationApproach = \"BBA\"},",
                 "\n                                                 };",
-                "\n",
+                "\nvar inputDataPaaGic = new GroupOfInsuranceContract[]{ sampleGic with {SystemName = \"Paa1\", LiabilityType = LiabilityTypes.LRC, ContractualCurrency = \"USD\", ValuationApproach = \"PAA\"},",
```

Comment:
Minor comment: use the consts instead of hardcoding strings, wherever applicable (for example here ValuationApproach).

Reply:
This is a valid point but the whole file requires this king of change which I then defer to a follow up PR. :)